src/structlog/processors.py

```diff
@@ -317,9 +319,9 @@ class JSONRenderer:
         serializer:
             A :func:`json.dumps`-compatible callable that will be used to
             format the string.  This can be used to use alternative JSON
-            encoders (default: :func:`json.dumps`).
-
-            .. seealso:: :doc:`performance` for examples.
+            encoders like `orjson <https://pypi.org/project/orjson/>`__ or
+            `RapidJSON <https://pypi.org/project/python-rapidjson/>`_
+            (default: :func:`json.dumps`).
```

Comment:
this is probably a merge artifact or something – please revert

Reply:
Done